### PR TITLE
Added Javascript fibonacci

### PR DESCRIPTION
**Pull request by penguinFSS**

### CODE REVIEW

**Comment by penguinFSS** [comment]
What's wrong with the check ?

**Comment by fharookshaik** [comment]
Hey Don't worry about the error check. It's a problem with one of the github actions. I'll review your PR.

Thanks for Contributing.

Happy Coding!!

File: JS_fibonacci.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function fibonacci(num) {
+  if (num <= 1) return 1;
+
+  return fibonacci(num - 1) + fibonacci(num - 2);
+}
```
Comment (fharookshaik) [inline JS_fibonacci.js:5]: Hey, Appreciated Your Effort.  But this repo need a complete program not just the executing part.
Review Your Code Once.

Happy Coding!!

Comment (fharookshaik) [reply JS_fibonacci.js:5]: The program needs to ask how many fibonacci terms should it needs to print.

Happy Coding!!

**Comment by penguinFSS** [comment]
Fixed. Now the program ask the user for input and it responds with the correct answer

**Review by fharookshaik — COMMENTED** [review]
I appreciate your effort. I see that the result is only that particular term printing out as alert. How about to console.log all the the terms upto the certain input terms. 

Please review Your code once.

Happy Coding!!

File: JS_fibonacci.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+function fibonacci(num) {
+  if (num <= 1) return 1;
```
Comment (fharookshaik) [inline JS_fibonacci.js:2]: Codacy found an issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288785)

**Comment by fharookshaik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- JS_fibonacci.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288785&bid=20460284)

**Comment by fharookshaik** [comment]
Please make your changes quickly before other's PR is pulled.

Thank You

**Comment by fharookshaik** [comment]
Since you haven't updated your code with the requested changes nor responded to the messages, your PR will be automatically closed after the month of October or else other's PR is merged before yours.

**Comment by fharookshaik** [comment]
I'm closing this PR as stipulated time for changes is over.
Thank You for your Conribution.

**Pull request closed without merging**